odoo/tools/view_validation.py

```diff
@@ -24,6 +24,7 @@
     'self',
     'uid',
     'context',
+    'companies',
     'context_today',
     'allowed_company_ids',
     'current_company_id',
```

Comment:
Should we depreciate these two values?

Reply:
Yes, but we will do it in a follow-up PR